packages/@atjson/offset-annotations/src/annotations/table-of-contents.ts

```diff
@@ -1,20 +1,20 @@
 import { BlockAnnotation } from "@atjson/document";
 
 /**
- * A table of content is designed to enhance the usability and navigation of
+ * Table of contents is designed to enhance the usability and navigation of
  * our content. This feature provides users with a structured and accessible
  * way to navigate through various sections of a story, especially long-form
  * articles, improving the overall user experience and engagement. It will
  * make it easier for users to find specific sections or information quickly.
  * It also encourages readers to explore more content by providing them easy
  * access to different parts of our content.
  *
- * Table of Content displays data as a list of links that tag to headers that
+ * Table of Contents displays data as a list of links that anchor to headers that
  * exist in the body of a story.
  *
  */
 
-export class TableOfContent extends BlockAnnotation<{
+export class TableOfContents extends BlockAnnotation<{
```

Comment:
Please update the comment for the `isCollapsible` property below - that is not meant to capture the initial state for the Table of Contents.

Also, in general, please remove abbreviations of `toc` in favor of spelling out "table of contents" in comments/variable names/etc